script/util/copy-verifier.sh

```diff
@@ -4,99 +4,48 @@ GIT_REPO_PATH=../mapreduce-plonky2
 CODE_DIR_PATH=groth16-framework/test_data
 ENV=$1
 VERIFIER_FOLDER=./script/output/$1
-VERIFIER_FILE=$VERIFIER_FOLDER/Groth16Verifier.sol
-VERIFIER_EXTENSIONS_FILE=$VERIFIER_FOLDER/Groth16VerifierExtensions.sol
-#VERIFIER_SOL_URL="https://pub-64a4eb6e897e425083647b3e0e8539a1.r2.dev/groth16_assets/verifier.sol"
-
+VERIFIER_FILE=$VERIFIER_FOLDER/Verifier.sol.ignore
+VERIFIER_EXTENSION_FILE=$VERIFIER_FOLDER/Groth16VerifierExtension.sol.ignore
+VERIFIER_EXTENSION_URL=https://raw.githubusercontent.com/Lagrange-Labs/mapreduce-plonky2/refs/heads/main/groth16-framework/test_data/Groth16VerifierExtension.sol
```

Comment:
We're changing the branch from `holesky` to `main` for the source of truth. I don't see any reason not to pull the contract from the `main` branch. Also grabbing it from github reduces the complexity of this script.